src/Model/CodeModelGo.cs

```diff
@@ -87,18 +87,17 @@ public IEnumerable<string> ModelImports
                     .ForEach(mt =>
                     {
                         mt.AddImports(imports);
-                        // include the strconv package if this response type has
-                        // response headers that need to be converted to integers
+                        // add any imports for packages needed to parse/convert response header values
                         if (mt.IsResponseType && mt.ResponseHeaders().Any())
                         {
-                            foreach (var p in mt.Properties)

```

Comment:
>Properties [](start = 49, length = 10)

What will happen with the properties?

Reply:
The imports for properties on composite types are handled on line 89; this loop was meant to add the imports for the header response types.